orchestrator/assets/commuting.py

```diff
@@ -0,0 +1,226 @@
+from datetime import datetime
+
+from dagster import (
+    asset,
+    get_dagster_logger,
+    ResourceParam,
+)
+from dagster_pandera import pandera_schema_to_dagster_type
+import pandera as pa
+from pandera.typing import Series, DateTime
+import pandas as pd
+
+from orchestrator.assets.utils import empty_dataframe_from_model, add_dhub_sync
+from orchestrator.resources.datahub import DataHubResource
+
+logger = get_dagster_logger()
+
+
+class CommutingSurvey(pa.SchemaModel):
```

Comment:
This is currently not used. Might want to remove it, as different year of survey has different format.